support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -315,10 +296,31 @@ export const productCatalogDescription: Record<string, ProductDescription> = {
 	},
 };
 
+export const productCatalogDescriptionAdditionalMissing: typeof productCatalogDescription =
+	{
+		...productCatalogDescription,
+		TierThree: {
+			label: productCatalogDescription.TierThree.label,
+			benefitsSummary: productCatalogDescription.TierThree.benefitsSummary,
+			benefits: productCatalogDescription.TierThree.benefits,
+			benefitsAdditional: productCatalogDescription.SupporterPlus.benefits,
+			deliverableTo: productCatalogDescription.TierThree.deliverableTo,
+			ratePlans: productCatalogDescription.TierThree.ratePlans,
+		},
+		Contribution: {
+			label: productCatalogDescription.Contribution.label,
+			benefits: productCatalogDescription.Contribution.benefits,
+			benefitsMissing: productCatalogDescription.SupporterPlus.benefits,
```

Comment:
is there not some overlap between benefits and missing benefits? Newsletter for example

Reply:
Ah yes,  your right for missing it wont work neatly, let me refactor